v23.1/restoring-backups-across-versions.md

```diff
@@ -16,17 +16,16 @@ Since CockroachDB considers the cluster version when running a backup, this page
 
 ## Support for restoring backups into a higher version
 
-Cockroach Labs supports restoring a backup taken on a cluster on a specific major version into a cluster that is on the same version or the next major version. Therefore, when you're upgrading your cluster from version N to N+1, if you took a backup on major version N, you can restore it to your cluster on either major version N or N+1.
+Cockroach Labs supports restoring a backup taken on a cluster on a specific major version into a cluster that is on the same version or the next major version. Therefore, when upgrading your cluster from version N to N+1, if you took a backup on major version N, you can restore it to your cluster on either major version N or N+1. Restoring backups outside major version N or N+1 is not supported. 
```

Comment:
We should also indicate here that customers should take a new backups everytime they upgrade to a new major version. Could we add this to the upgrade documentation if it isn't there already?

Reply:
I think we will want to add about recommending a backup prior to upgrading to this page to @ashleykwtam, perhaps you can add the sentence or two to an include and place on this page and the upgrade page.

Reply:
Added recommending a backup prior to the restoring. I added the paragraph to the intro block, but perhaps it should be it's own heading? Since this page discusses restores mainly I don't think we should have a separate backup heading

Reply:
We could highlight it as a note? Or a sub-heading?